bigflow/__init__.py

```diff
@@ -26,3 +26,9 @@
     __all__.append('monitoring')
 except ImportError:
     pass
+
+try:
+    from . import dataflow
+    __all__.append('dataflow')
```

Comment:
IMO we should not add 'dataflow' to `__all__`
It is required only to support cases like:

```python
from bigflow import *
my_job = dataflow.BeamJob(...)
```

it is better to write
```python
import bigflow
my_job = bigflow.dataflow.BeamJob(...)
```
but the best option is to explicitly import needed module:

```python
from bigflow.dataflow import BeamJob
my_job = BeamJob(...)
```
or
```python
import bigflow.dataflow
my_job = bigflow.dataflow.BeamJob(...)
```

In such case there is no need to add anything to `bigflow/__init__.py` at all

Reply:
done